misk/src/main/kotlin/misk/web/actions/WebActionMetadataAction.kt

```diff
@@ -0,0 +1,26 @@
+package misk.web.actions
+
+import misk.security.authz.Unauthenticated
+import misk.web.Get
+import misk.web.RequestContentType
+import misk.web.ResponseContentType
+import misk.web.jetty.WebActionsServlet
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class WebActionMetadataAction : WebAction {
+  @Inject internal lateinit var servlet: WebActionsServlet
```

Comment:
mind injecting this as a `Provider<WebActionsServlet>` ? As-is there’s a little dependency cycle and this seems like the most natural place to break it.

```
@Inject internal lateinit var servletProvider: Provider<WebActionsServlet>
```